src/node_http_parser.cc

```diff
@@ -90,7 +91,7 @@ struct StringPtr {
   // to leak references. See issue #2438 and test-http-parser-bad-ref.js.
   void Save() {
     if (!on_heap_ && size_ > 0) {
-      char* s = new char[size_];
+	  char *s = (char *)ikmem_malloc(size_);
```

Comment:
Could you remove the style (tab + `*` location) changes?

Also, is it necessary to cast `ikmem_malloc` calls? I know it isn't recommended to cast `malloc` calls.